apps/backend/review/src/createReview.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from "express";
+import { IDatabase } from "pg-promise";
+
+// POST /api/reviews - Create a new review
+const createReview = async (
+  req: Request,
+  res: Response,
+  db: IDatabase<object>,
+) => {
+  // TODO: AUTHENTICATION
+  const userID = 1; // placeholder for authenticated user ID
+  const { stars, comment, listingID } = req.body;
+
+  if (!stars || !comment || !listingID) {
+    console.error("missing parameter in request");
+    return res.status(400).json({ error: "missing parameter in request" });
+  }
```

Comment:
Do we need any additional validation here? E.g. stars cannot be > 5, etc?

Reply:
Nope. Swaggerhub has no error defined for invalid stars, and is impossible to click over 5 stars

Reply:
Server side validation for every property is required, frontend validation doesn't stop someone from making a network request
```console
curl -X POST -H "Content-Type: application/json" -d '{"stars": 1000, "comment": "", "listingId": 1}' http://localhost/api/review
```

Reply:
gotchya, updated!